extensions/amp-story/0.1/amp-story.js

```diff
@@ -295,6 +298,17 @@ export class AmpStory extends AMP.BaseElement {
   }
 
   /** @private */
+  lockBody_() {
+    const document = this.win.document;
+    setImportantStyles(document.documentElement, {
```

Comment:
Any reason we can't just add this to `amp-story.css` instead of adding it in the body?

Reply:
actually i had a question regarding about this specific part, is there any plan at all where we allow `amp-story` without `standalone`? 

If no, css is much better.

If yes/maybe, a conditional check of `standalone` attribute might and js based solution might be better